pravega-client-examples/src/main/java/io/pravega/example/streamprocessing/README.md

```diff
@@ -0,0 +1,177 @@
+# Pravega Stream Processing Example
+
+
+# Overview
+
+The examples in this directory are intended to illustrate
+how exactly-once semantics can be achieved with Pravega.
+To make it clear to the reader, the entirety of the stream processor is defined
+in a single source file of under 500 lines.
+In particular, these illustrative examples do not use Apache Flink.
+
+As one reads and understands this example exactly-once processor, they should
```

Comment:
This paragraph is trying to say too many things at the same time.

Reply:
I cleaned this up.